server/modlog.ts

```diff
@@ -497,59 +500,60 @@ export class Modlog {
 		if (rooms.includes('global')) roomChecker = `(roomid LIKE 'global-%' OR ${roomChecker})`;
 
 		let query = `SELECT *, (SELECT group_concat(userid, ',') FROM alts WHERE alts.modlog_id = modlog.modlog_id) as alts FROM modlog`;
+		if (this.useFTS) query += ` LEFT JOIN modlog_fts ON modlog_fts.rowid = modlog_id`;
 		query += ` WHERE ${roomChecker}`;
 
 		if (search.anyField) {
-			query += ` AND action LIKE '%' || ? || '%'`;
+			query += ` AND action LIKE ? || '%'`;
```

Comment:
Is this satisfactory? We have a lot of actions that are a combination of two words, like `GTS GIVEAWAY`. Would they expect a search on "giveaway" to return a GTS GIVEAWAY row?

Reply:
I'll post in WAIL.